automated-comments/csharp/general/do_not_assign_and_return.md

```diff
@@ -0,0 +1 @@
+Consider returning the value directly instead of storing it into an intermediate value and then returning it.
```

Comment:
Will the analyser still block if this change is not made? Or is this always a "tip"? If it still blocks:

```suggestion
Return the value directly, instead of storing it into an intermediate value and then returning it.
```

Reply:
The analyser will not block (this is used in the approve_with_comment) path.